Makefile

```
@@ -172,6 +172,21 @@ install-python-linter:
 python-lint:
 	scripts/lint.sh
 
+## install-python-formatter: Install python formatter
+install-python-formatter:
+	pip install black==20.8b1
```

```suggestion
	pip install black
```
I suppose it's better don't have a hardcoded version, especially when it's beta.

> **NOTE: This is a beta product**
What this means for you is that until the formatter becomes stable, you should expect some formatting to change in the future. That being said, no drastic stylistic changes are planned, mostly responses to bug reports.

What's your opinion?

---

1. Why hardcoded version here is bad?
2. Especially because it's beta and the package has no guarantees about backward compatibility we should have a hardcoded version here.

---

I thought it would be better to, at least, upgrade to a stable version. And if there were mistakes in it, it would be great to have them fixed.

---

We will upgrade it manually because of (2).
In general, a lot of software that claims that it's beta is much more stable than other ones that claim that their software is "stable".
So, "beta" here means that there's not backward compatibility garantuees. Because of this it's better to upgrade it manually with having hardcoded version.

---

Oh, OK then.